integration_test/third_party_apps_data/applications/mongodb/debian_ubuntu/install

```diff
@@ -0,0 +1,42 @@
+set -e
+
+sudo apt-get install -y gnupg wget
+wget -qO - https://www.mongodb.org/static/pgp/server-5.0.asc | sudo apt-key add -
+
+source /etc/os-release
+case $VERSION_ID in
+  20.04)
+    echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu focal/mongodb-org/5.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-5.0.list
+    ;;
+  18.04)
+    echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/5.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-5.0.list
+    ;;
+  16.04)
+    echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu xenial/mongodb-org/5.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-5.0.list
+    ;;
+  # Debian 11 is not officially supported, so use 'buster'.
+  11)
+    echo "deb http://repo.mongodb.org/apt/debian buster/mongodb-org/5.0 main" | sudo tee /etc/apt/sources.list.d/mongodb-org-5.0.list
+    ;;  
+  10)
+    echo "deb http://repo.mongodb.org/apt/debian buster/mongodb-org/5.0 main" | sudo tee /etc/apt/sources.list.d/mongodb-org-5.0.list
+    ;;
+  9)
+    echo "deb http://repo.mongodb.org/apt/debian stretch/mongodb-org/5.0 main" | sudo tee /etc/apt/sources.list.d/mongodb-org-5.0.list
+    ;;
+  *)
+    echo -n "unknown version"
+    exit 1
+    ;;
+esac
+
+sudo apt-get update
+sudo apt-get install -y \
```

Comment:
Is there anything we can do to test the other supported versions of mongodb?

Reply:
Hmm we could try to specify the default version of mongodb per os-release... however I've been burnt by not pinning versions. We could arbitrarily install versions on certain releases of debian/ubuntu but then the test becomes less focused and harder to debug potentially. Perhaps something we can solve later? I'm open to opinions or further enhancements. Have any thoughts @quentinmit @qingling128 ?

Reply:
Yeah, I think this is OK for the first iteration.

Separately, we should have some discussion about testing different versions of the application. But better punt that until the effort to reduce flakiness lands (aka reduce the scope of tests that are triggered based on what files the PR touches). With the current setup, if we try to test all supported app version on all distros for all PRs, the flakiness and quota issues are gonna bite us hard. It also feels like an overkill to test all the combinations.

CC @martijnvans for visibility in case anything is not accurate.

Reply:
+1, fine to punt for this PR and think about for the future.